traveler-swift-core/TravelerKit/Models/ProductItemSort.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  ProductItemSort.swift
+//  TravelerKit
+//
+//  Created by Rakin Hoque on 2019-11-18.
+//  Copyright © 2019 Ata Namvari. All rights reserved.
+//
+
+import Foundation
+
+/// Represents the different sorting options a `BookingItemQuery` can have
+public enum ProductItemSortOption: String, Decodable {
```

Comment:
If this sorting option only pertains to BookingItems then it should be called BookingItemSortOption. If the sort option could be used in all products then the documentation should say that it represents the different sorting options a `Product`can have. My guess is that it is the latter.

Reply:
Discussed with backend and RE and the goal is to have it as something that conforms across products, so I've updated the comments.